core/market/src/protocol/negotiation/provider.rs

```diff
@@ -170,7 +170,7 @@ impl NegotiationApi {
         caller: String,
         msg: ProposalReceived,
     ) -> Result<(), CounterProposalError> {
-        log::debug!(
+        log::trace!(
```

Comment:
Do we log anywhere, that we received Proposal from other party?

Reply:
We had this information tripled
```
[2020-11-05T21:41:06Z DEBUG ya_net::service] Incoming msg from = 0xe0489275bd2f5fce6ead8256382615213087bef7, to = /net/0xb11cd0514d0b33251345b12387cb893a0d75f1f5/market/protocol/mk1/negotiation/requestor/proposal/ProposalReceived, fwd to local addr = /public/market/protocol/mk1/negotiation/requestor/proposal/ProposalReceived, request_id: 6340869910067749
[2020-11-05T21:41:06Z DEBUG ya_market::protocol::negotiation::requestor] Negotiation API: Received proposal [P-a6e9fdd7669bbeb383af0b2326bd1dd20fd4a1ea28d411f7ea78f5827186374b] from [0xe0489275bd2f5fce6ead8256382615213087bef7].
[2020-11-05T21:41:06Z INFO  ya_market::negotiation::common] Received counter Proposal [R-a6e9fdd7669bbeb383af0b2326bd1dd20fd4a1ea28d411f7ea78f5827186374b] for Proposal [R-8ace9eaf3a236b36c97249c8714076c575a4c9478824e7a73d738f3cffed1e3c] from [0xe0489275bd2f5fce6ead8256382615213087bef7].
```

Thats why I've changed DEBUG to TRACE for first two of them